cmd/soroban-cli/src/commands/network/mod.rs

```diff
@@ -89,8 +89,8 @@ pub enum Error {
     FailedToParseJSON(String, serde_json::Error),
     #[error("Invalid URL {0}")]
     InvalidUrl(String),
-    #[error("Inproper response {0}")]
-    InproperResponse(String),
+    #[error("Improper response {0}")]
+    ImproperResponse(String),
```

Comment:
I think this incorrectly named in multiple places, so I'd not fix in this PR if we are not fix in all the places

Reply:
Hello @janewang, I did a search, and this is the only occurrence of this typo